src/librustdoc/html/render/print_item.rs

```diff
@@ -647,6 +647,27 @@ fn item_function(cx: &Context<'_>, it: &clean::Item, f: &clean::Function) -> imp
     })
 }
 
+/// Struct used to handle insertion of "negative impl" marker in the generated DOM.
+///
+/// This marker appears once in all trait impl lists to divide negative impls from positive impls.
```

Comment:
nit:


```suggestion
/// This marker may appear at most once in every trait impl lists to divide negative impls from positive impls.
```

It might not appear, if item has only negative impls, I think?
And "in all trailt impl lists" makes it sound like it's only gonna appear once on a single HTML page.